### DISCUSSION TITLE
Rule BR-BT-00052-0022

### DISCUSSION THREAD

**Discussion by StephaneElGuedj** [OP-TED/eForms-SDK#1020] [Q&A] [answered]
Hello !
I think there is a problem with rule BR-BT-00052-0022:

The rule message indicates: 'The procedure will take place in successive stages. At each stage, some participants may be eliminated' (BT-52-Lot) is mandatory in notice type '16 – Contract notice – general directive, standard regime' when: 'Type of procedure' (BT-105) must not be either 'Competitive dialogue', 'Innovation partnership'; 'Negotiated with prior publication of a call for competition / competitive with negotiation'

Whereas the XPATH rule indicates (no error if): 
count(cbc:CandidateReductionConstraintIndicator) > 0 or _**not**_(../../cac:TenderingProcess/cbc:ProcedureCode/normalize-space(text()) = ('comp-dial','innovation','neg-w-call'))

Correct me if I'm wrong, but the XPATH rule corresponding to the rule message should be ("not" removed after the "or"): 
count(cbc:CandidateReductionConstraintIndicator) > 0 or (../../cac:TenderingProcess/cbc:ProcedureCode/normalize-space(text()) = ('comp-dial','innovation','neg-w-call'))

What is the correct information ? The rule message or the XPATH check ?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This question has already reached our helpdesk and is currently being considered.
BT-52-Lot only makes sense for multi-stages procedures and may only be required for these ones.
The rule expression is correct and the associated message needs some rework.
